pinot-spi/src/main/java/org/apache/pinot/spi/config/table/DedupConfig.java

```diff
@@ -52,4 +61,8 @@ public boolean isDedupEnabled() {
   public String getMetadataManagerClass() {
     return _metadataManagerClass;
   }
+
+  public double getMetadataTTL() {
```

Comment:
this won't work for dedupManager since there is no support for removeExpiredPrimaryKey functionalities in the dedupPartitionManager.

Reply:
do we plan to add this feature to dedupPartitionManager as well?

Reply:
Yes, initially I plan to add it in another PR. But let me put the logic in together.